proto/resource_manager.proto

```diff
@@ -28,6 +28,7 @@ service ResourceManager {
 
 message ListResourceGroupsRequest{
   bool with_ru_stats = 1;
+  uint32 keyspace_id = 2;
```

Comment:
For tikv/tiflash, they need to get all resource groups of all keyspaces, how to express this?